components/UI/iconsComponents/icons/closeIcon.tsx

```diff
@@ -1,8 +1,12 @@
 import React, { FunctionComponent } from "react";
 
-const CloseIcon: FunctionComponent = () => {
-  return (
-    <svg
+interface CloseIconProps {
+  onClick?: () => void; 
```

Comment:
Dont put the onclick inside the icon component. Instead you can wrap the icon component with a div in your other files
```tsx
<div onlick=...>
    <CloseIcon... />
</div>
```